src/resolution_functions/instrument.py

```diff
@@ -618,52 +645,125 @@ def get_model_signature(self, model_name: Optional[str] = None) -> Signature:
+
         Returns
         -------
         available_models
             A list of all available models.
         """
-        return list(self._models.keys())
+        return list(self._models)
 
     @property
-    def available_models_and_configurations(self) -> dict[str, list[str]]:
+    def available_unique_models(self) -> list[str]:
         """
-        A dictionary mapping each available :term:`model` to the user
-        :term:`configurations<configuration>` available for that :term:`model`.
+        A list of all unique :term:`models<model>` available for this :term:`version` of this
+        :term:`instrument`.
 
-        All :term:`models<model>` available for this :term:`version` of this :term:`instrument`,
-        and all their :term:`configurations<configuration>` are listed.
+        Only includes the versioned models, i.e. lists all versions of all models, but does not list
+        the recommended models.
 
         Returns
         -------
-        models_and_configurations
-            All models and all their configurations.
+        available_models
+            A list of all available unique models.
         """
-        return {model_name: list(model['configurations'].keys())
-                for model_name, model in self._models.items()}
+        return [name for name, value in self._models.items() if not isinstance(value, str)]
 
-    @property
-    def all_available_models_options(self) -> dict[str, dict[str, list[str]]]:
+    def format_available_models_and_configurations(self) -> str:
+        """
+        Formats all available :term:`models<model>` and :term:`configurations<configuration>` into a
+        table.
+
+        The table shows each :term:`model` and either which other :term:`model` it is an alias for,
+        or all the :term:`configurations<configuration>` required by that :term:`model`.
+
+        Returns
+        -------
+        str
+            A string containing the nicely formatted table.
         """
-        A dictionary mapping each available :term:`model`, to the user
-        :term:`configurations<configuration>` and their :term:`options<option>`.
+        contents = [['MODEL', 'ALIAS FOR', 'CONFIGURATIONS']]
 
-        All :term:`models<model>` available for this :term:`version` of this :term:`instrument`, all
-        the :term:`configurations<configuration>` of each of the :term:`models<model>`, and all the
-        :term:`options<option>` for each of the :term:`configurations<configuration>`, are listed.
+        for model_name, model_data in self._models.items():
+            if isinstance(model_data, str):
+                contents.append([model_name, model_data, ''])
+            else:
+                for i, config_name in enumerate(model_data['configurations']):
+                    if i == 0:
+                        contents.append([model_name, '', config_name])
+                    else:
+                        contents.append(['', '', config_name])
```

Comment:
```suggestion
                tmp = [['', '', config_name]
                       for config_name in model_data['configurations']]
                tmp[0][0] = model_name
                contents.extend(tmp)
```
Is this more legible or less?